data/xml/Q17.xml

```diff
@@ -216,102 +232,124 @@
-    <author><first>Diarmuid</first><last>Ó Séaghdha</last></author>
+    <author><first>Diarmuid
+                            Ó</first><last>Séaghdha</last></author>
     <author><first>Ira</first><last>Leviant</last></author>
     <author><first>Roi</first><last>Reichart</last></author>
     <author><first>Milica</first><last>Gašić</last></author>
     <author><first>Anna</first><last>Korhonen</last></author>
     <author><first>Steve</first><last>Young</last></author>
     <year>2017</year>
-    <abstract>We present Attract-Repel, an algorithm for improving the semantic quality of word vectors by injecting constraints extracted from lexical resources. Attract-Repel facilitates the use of constraints from mono- and cross-lingual resources, yielding semantically specialized cross-lingual vector spaces. Our evaluation shows that the method can make use of existing cross-lingual lexicons to construct high-quality vector spaces for a plethora of different languages, facilitating semantic transfer from high- to lower-resource ones. The effectiveness of our approach is demonstrated with state-of-the-art results on semantic similarity datasets in six languages. We next show that Attract-Repel-specialized vectors boost performance in the downstream task of dialogue state tracking (DST) across multiple languages. Finally, we show that cross-lingual vector spaces produced by our algorithm facilitate the training of multilingual DST models, which brings further performance improvements.</abstract>
-    <href>https://transacl.org/ojs/index.php/tacl/article/download/1171/246</href>
+    <month>December</month>
+    <doi>10.1162/tacl_a_00063</doi>
+    <abstract>We present Attract-Repel, an algorithm for improving the
+                    semantic quality of word vectors by injecting constraints extracted from lexical
+                    resources. Attract-Repel facilitates the use of constraints
+                    from mono- and cross-lingual resources, yielding semantically specialized
+                    cross-lingual vector spaces. Our evaluation shows that the method can make use
+                    of existing cross-lingual lexicons to construct high-quality vector spaces for a
+                    plethora of different languages, facilitating semantic transfer from high- to
+                    lower-resource ones. The effectiveness of our approach is demonstrated with
+                    state-of-the-art results on semantic similarity datasets in six languages. We
+                    next show that Attract-Repel-specialized vectors boost
+                    performance in the downstream task of dialogue state tracking (DST) across
+                    multiple languages. Finally, we show that cross-lingual vector spaces produced
+                    by our algorithm facilitate the training of multilingual DST models, which
+                    brings further performance improvements.</abstract>
```

Comment:
Can you reflow this ("\n" → " ")?

Reply:
If I had a dollar for every inconsistency and special case in the input XML...

Reply:
Yes, you and @davidweichiang both. 

I can’t give you dollars, but can do lunch :)